frontend/src/scenes/dashboard/DashboardHeader.tsx

```diff
@@ -239,6 +239,7 @@ export function DashboardHeader(): JSX.Element | null {
                                     <LemonButton
                                         type="secondary"
                                         data-attr="dashboard-share-button"
+                                        data-tooltip="experiment-new-dashboard-product-tour-1"
```

Comment:
Thinking to the future... We use `data-attr` everywhere and encourage people to do so for `autocapture`. Might be good to have that as a selectable option (thinking more about productizing the feature)